src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -58,21 +52,6 @@ public HelpWindow(Stage root) {
         titleLabel.getStyleClass().add("help-title");
         mainContainer.getChildren().add(0, titleLabel);
         initializeCommandTable();
-        helpMessageWebView.getEngine().loadContent(
-                "<html><body style='display: flex; justify-content: center; align-items: center; height: 100%; "
-                        + "margin: 0; padding: 0; background-color: #2b2b2b;'>"
-                        + "<p style='color: #e6e6e6; font-family: \"Segoe UI\", sans-serif; font-size: 14pt; "
-                        + "text-align: center;'>"
-                        + "For more information, please refer to the user guide."
```

Comment:
I think the banner(?) on the UI saying "For more information..." above the button is quite useful and make the help page more user friendly. Can consider bringing it back!
<img width="1037" alt="Screenshot 2567-11-04 at 17 38 43" src="https://github.com/user-attachments/assets/aab52e39-8406-4276-8b50-acda454be749">